source/retryable-writes/retryable-writes.rst

```diff
@@ -791,14 +795,16 @@ Why does the driver only add the RetryableWriteError label to errors that occur
 
 The driver does this to maintain consistency with the MongoDB server.
 Servers that support the RetryableWriteError label (MongoDB version 4.4 and newer)
-only add the label to an error when the client has added a txnNumber to the 
+only add the label to an error when the client has added a txnNumber to the
 command, which only happens when the retryWrites option is true on the client.
 For the driver to add the label even if retryWrites is not true would be
 inconsistent with the server and potentially confusing to developers.
 
 Changes
 =======
 
+2022-01-07: Note that drivers should retry handshake network failures.
```

Comment:
Reminder to bump version and last modified date in header.

Reply:
done!